content/20.zksync-protocol/20.contracts/31.evm-emulator/05.limitations.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Limitations and Known Issues
```

Comment:
This page basically duplicates "Differences Between the EVM Emulator and Ethereum" page. I propose to remove it (probably it was created by AI, page is a bit confusing).

Reply:
yeah, there's some overlap like the precompiles and opcodes but other things are only listed in this page, like access lists, refunds, etc. But as you mentioned above, if we have a specific page for all gas related feaures/limitations/differences, I agree we can delete this page.